streams/src/main/java/org/apache/kafka/streams/kstream/internals/StatefulRepartitionNode.java

```diff
@@ -21,42 +21,33 @@
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.common.utils.Bytes;
-import org.apache.kafka.streams.processor.ProcessorSupplier;
 import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder;
 import org.apache.kafka.streams.state.KeyValueStore;
 
 class StatefulRepartitionNode<K, V, T> extends RepartitionNode<K, V> {
 
-    private final ProcessorSupplier<K, Change<V>> statefulProcessorSupplier;
     private final MaterializedInternal<K, T, KeyValueStore<Bytes, byte[]>> materialized;
 
-    StatefulRepartitionNode(final String parentProcessorNodeName,
-                            final String processorNodeName,
+    StatefulRepartitionNode(final String nodeName,
```

Comment:
Meta comment: let's distinguish the names more explicitly please... the `sourceName` and the `nodeName` are really for different types of names right? For example, try `nodeName` and `repartitionTopicName`.

Reply:
For now I'm trying to keep the names the same as they are generated inside KStreamImpl, KTableImpl etc, makes it easier to reason where the information came from.  But we can discuss some more.